frodo.js

```diff
@@ -1,4 +1,4 @@
-const objectValues = Object.values(object);
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.